project/AppDependencies.scala

```diff
@@ -5,14 +5,17 @@ import sbt._
 
 object AppDependencies {
 
+  val bootstrapVersion = "5.24.0"
```

Comment:
SHouldn't this be 6.4.0 and be used below?

Reply:
yes, I missed that , will do it